src/hooks/video/agora/useAgoraCamera.ts

```diff
@@ -31,9 +31,7 @@ export const useAgoraCamera: ReactHook<
   } = useShowHide();
 
   const toggleCamera = () => {
-    if (!localCameraTrack) return;
-
-    localCameraTrack.setEnabled(!isCameraOn);
+    localCameraTrack?.setEnabled(!isCameraOn);
     setIsCameraOn(!isCameraOn);
   };
```

Comment:
`useShowHide` already has a premade `toggle` function that is more efficient than this, uses proper `useState` update patterns, and is already `useCallback`'d

Reply:
fixed in 24b3cd8ba03f64a21bea2d35a249cf817facc979